packages/dds/tree/src/test/shared-tree/undo.spec.ts

```diff
@@ -673,6 +675,78 @@ describe("Undo and redo", () => {
 			"Error: Unable to revert a revertible that has been disposed.",
 		);
 	});
+
+	it("clone list of revertibles", () => {
+		const view = createInitializedView();
+		const { undoStack } = createTestUndoRedoStacks(view.events);
+
+		assert(view.root.child !== undefined);
+		view.root.child.propertyOne = 256; // 128 -> 256
+		view.root.child.propertyTwo.itemOne = "newItem"; // "" -> "newItem"
+
+		const forkedView = view.fork();
+
+		const batchedRevertibles: RevertibleAlpha[] = [];
+		for (const revertible of undoStack) {
+			batchedRevertibles.push(revertible);
+		}
```

Comment:
Is there a reason not to just do `const batchedRevertibles = [...undoStack];`

Reply:
Spread operator also works. Thanks!